variants/STM32C0xx/C092CBT_C092RBT_C092RC(I-T)/PeripheralPins_NUCLEO_C092RC.c

```diff
@@ -0,0 +1,412 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020, STMicroelectronics
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+/*
+ * Automatically generated from STM32C092R8Tx.xml, STM32C092RBTx.xml
+ * CubeMX DB release 6.0.120
```

Comment:
Seems not be a copy of the PeripheralPins.c version should be 6.0.140

Reply:
I honestly do not know how this one should have been generated. I had copied over the one from C071-RB first, but then copied the contents of the generic variant of the C092 with the necessary edits but not the header. So it's edited manually, which is probably not the way it should be 😅.

In any case, how is this file normally produced? Couldn't find it in the wiki.

Reply:
The custom file is a copy of the one in the generic folder then edited manually to remove some pins.

Reply:
I remove it as the one for C071 has several diff. C071 custom peripheralPins is used to remove some USB pins. C092 does not have USB.